src/utils/bcrypt.ts

```diff
@@ -0,0 +1,7 @@
+import * as bcrypt from 'bcrypt';
+
+export const encodePassword = (rawPassword: string) => {
+  const SALT = bcrypt.genSaltSync(10);
```

Comment:
**suggestion**: Consider making the salt rounds configurable via environment variables for flexibility.